src/kit/LineChart.tsx

```diff
@@ -113,18 +113,19 @@ export const LineChart: React.FC<LineChartProps> = ({
     });
 
     return [xValues, ...yValuesArray];
-  }, [series, xAxis]);
+  }, [series, xAxis, hiddenSeries]);
 
-  const xTickValues: uPlot.Axis.Values | undefined = useMemo(
-    () =>
-      xAxis === XAxisDomain.Time &&
-      chartData.length > 0 &&
-      chartData[0].length > 0 &&
-      chartData[0][chartData[0].length - 1] - chartData[0][0] < 43200 // 12 hours
-        ? getTimeTickValues
-        : undefined,
-    [chartData, xAxis],
-  );
+  const xTickValues: uPlot.Axis.Values | undefined = useMemo(() => {
+    if (xAxis === XAxisDomain.Time) {
+      const timeDelta = xRange?.[XAxisDomain.Time]
+        ? (xRange[XAxisDomain.Time]?.[1] || 0) - (xRange[XAxisDomain.Time]?.[0] || 0)
```

Comment:
I think using `??` is better even though its the same meaning in this case

Reply:
✅